### DISCUSSION TITLE
[Package Request] TaskbarX

### DISCUSSION THREAD

**Discussion by SuntyX** [microsoft/winget-pkgs#4086] [General] [answered]
Is it possible to add TaskbarX to winget ?

https://github.com/ChrisAnd1998/TaskbarX

**Comment by KaranKad** [comment, marked as answer]
TaskbarX download link has the application inside a zip archive and winget currently cannot extract the installer from zip archive and install it. [winget-cli/issues/140](https://github.com/microsoft/winget-cli/issues/140)

**Reply by SuntyX** [reply]
ok thx for the answer

**Reply by roke-julian-lockhart** [reply]
https://github.com/ChrisAnd1998/TaskbarX/issues/403#issuecomment-1838848987
